README.md

```diff
@@ -105,6 +105,49 @@ Here is an illustration to show you what happens when this order is used:
 
 ![this](https://docs.google.com/drawings/d/1Ea_QJHo-9p7YW8l_by7S4NID0e-AGpXRzzitAlYY5Cc/pub?w=960&h=720)
 
+### Queue interop usage
+
+Swarrot supports [queue-interop](https://github.com/queue-interop/queue-interop) compatible MQ transports. 
+It supports both sides: message publishing as well as their consumption. 
+To start using it you have to find and install the queue-interop compatible library of a MQ broker you want to work with. 
+In the library doc find out how to create its context. 
+
+Send a message:
+
+```php
+<?php
+use Swarrot\Broker\MessagePublisher\InteropMessagePublisher;
+use Swarrot\Broker\Message;
+
+/** @var \Interop\Queue\PsrContext $context */
+
+$messagePublisher = new InteropMessagePublisher($context, 'aTopic');
+
+$messagePublisher->publish(new Message('aBody'));
+```
+
+_**Note: ** The second argument `key` is not supported at the moment._
```

Comment:
Maybe the place to also inform people that RetryProcessor is not supported by queue-interop publisher ?